andrewtpoe/react-companies
# adds architecture and placeholder

## ISSUE

none

## DESCRIPTION

This PR establishes some baseline architecture and adds placeholder components.

## WHERE SHOULD THE REVIEWER START?

`src/setup/index.js`

## HOW CAN THIS BE TESTED?

Pull the branch.
Run `yarn install`.
Run `yarn start`.
Visit http://localhost:3000
Observe that the site renders a simple placeholder.

## BREAKING CHANGES

none

## ANY NEW DEPENDENCIES ADDED?

several.

## CHECKLIST

_Be sure all items are_ ✅ _before submitting a PR for review._

* [x] I rebased this branch on the `master` branch
